python/extensions/python.bzl

```diff
@@ -88,14 +129,39 @@ Please define a toolchain that has is_version set to True.""")
  module has a toolchain of the same name.""".format(toolchain_attr.name))
             continue
         toolchain_names.append(name)
-        _python_register_toolchains(toolchain_attr, True)
-
-    # Create the hub for the interpreters and the
-    # the default toolchain.
+        toolchains.append(
```

Comment:
Hm, using the name to determine equality isn't sufficient. And we'll be removing the name attribute, so it won't be an option anyways. It should really be based on the settings of the toolchain. Similarly, whether we need to warn or not depends on if what was requested vs what is going to be used are the same or not.

Anyways -- please put a todo to note this. We can clean it up in a separate PR.

Reply:
I will put all of this in an issue.